helm/nifikop/templates/deployment.yaml

```diff
@@ -79,10 +79,10 @@ spec:
         - command:
             - /manager
           args:
-            - metrics-bind-address={{ .Values.metrics.port }}
-            - --leader-elect
+            - -metrics-bind-address=:{{ .Values.metrics.port }}
```

Comment:
I know the defaults for these port properties have colons in them, but are we sure that the port configuration works with and without the colon?

Reply:
`.Values.metrics.port` has to be a number as we use `containerPort: {{ .Values.metrics.port }}` which expects a number.

When having no colon in `metrics-bind-address` the operator fails with `{"level":"error","time":"2022-10-25T11:59:25.008+0300","caller":"nifikop/main.go:88","msg":"unable to start manager","error":"error listening on 8089: listen tcp: address 8089: missing port in address","stacktrace":"main.main\n\t/home/marcel/workspace/sick/git/github/nifikop/main.go:88\nruntime.main\n\t/usr/lib/go-1.19/src/runtime/proc.go:250"}`

Reply:
Thanks for the clarification